### PR TITLE
add helper function canBurnSynths() 

### PR DESCRIPTION
**Pull request by jacko125**
add helper function canBurnSynths()

### CODE REVIEW

File: contracts/Issuer.sol [hunk 1]
```diff
@@ -42,6 +42,12 @@ contract Issuer is MixinResolver {
         return IssuanceEternalStorage(resolver.requireAndGetAddress("IssuanceEternalStorage", "Missing IssuanceEternalStorage address"));
     }
 
+    /* ========== VIEWS ========== */
+
+    function canBurnSynths(address account) public view returns (bool) {
```
Comment (jacko125) [inline contracts/Issuer.sol]: @clementbalestrat Clem this function should help the dapp check if burnSynths is open for user.

Comment (justinjmoses) [reply contracts/Issuer.sol]: This can be `external` and also go in `IIssuer.sol`

File: contracts/Issuer.sol [hunk 2]
```diff
@@ -114,7 +120,7 @@ contract Issuer is MixinResolver {
         external
         onlySynthetix
     {
-        require(now >= lastIssueEvent(from).add(minimumStakeTime), "Minimum stake time not reached");
+        require(canBurnSynths(from), "Minimum stake time not reached");
```
Comment (justinjmoses) [inline contracts/Issuer.sol]: it's a little more gas to use this as a `public` function rather than an internal

File: contracts/Issuer.sol (hunk 1 above)
Comment (jacko125) [reply contracts/Issuer.sol]: Done. public so that the dapp can use the function.

File: contracts/Issuer.sol [hunk 3]
```diff
@@ -281,5 +287,5 @@ contract Issuer is MixinResolver {
 
     /* ========== EVENTS ========== */
 
-    event minimumStakeTimeUpdated(uint minimumStakeTime);
+    event MinimumStakeTimeUpdated(uint minimumStakeTime);
```
Comment (jacko125) [inline contracts/Issuer.sol]: comment from Kyle (iosiro)

**Pull request merged**